public/app/features/manage-dashboards/utils/validation.ts

```diff
@@ -50,6 +52,10 @@ export const validateUid = (value: string) => {
   return getDashboardAPI()
     .getDashboardDTO(value)
     .then((existingDashboard) => {
+      if (isDashboardResource(existingDashboard)) {
```

Comment:
I think this was a leftover of the older approach, right? if we want to use the new schema, we should pass `v2` to `getDashboardAPI` 

Do you want to tackle this in another PR? @dprokop

Reply:
Hey dom 👋🏾 , I am seeing `return getDashboardAPI().getDashboardDTO`,  without any parameters, this will never return a dashboard resource, no?

Reply:
ah yea, you right. Pushed. But it surfaces a problem with the api client, see https://github.com/grafana/grafana/pull/96666#discussion_r1888563599